bittensor/core/subtensor_api/__init__.py

```diff
@@ -0,0 +1,220 @@
+from typing import Optional, Union, TYPE_CHECKING
+
+from bittensor.core.async_subtensor import AsyncSubtensor as _AsyncSubtensor
+from bittensor.core.subtensor import Subtensor as _Subtensor
+from .chain import Chain as _Chain
+from .commitments import Commitments as _Commitments
+from .delegates import Delegates as _Delegates
+from .extrinsics import Extrinsics as _Extrinsics
+from .metagraphs import Metagraphs as _Metagraphs
+from .neurons import Neurons as _Neurons
+from .queries import Queries as _Queries
+from .stakes import Stakes as _Stakes
+from .subnets import Subnets as _Subnets
+from .utils import add_legacy_methods as _add_classic_fields
+from .wallets import Wallets as _Wallets
+
+if TYPE_CHECKING:
+    from bittensor.core.config import Config
+
+
+class SubtensorApi:
+    """Subtensor API class.
+
+    Arguments:
+        network: The network to connect to. Defaults to `None` -> `finney`.
```

Comment:
```suggestion
        network: The network to connect to. Defaults to `None` -> "finney".
```
Backticks should be used to denote valid Python.